### PR TITLE
Feature/netbox 2.6

### PR DESCRIPTION
**Pull request by corny**

### CODE REVIEW

**Comment by madeinoz67** [comment]
could we get the workaround for #51 incorporated you look at merging?

File: README.md [hunk 1]
```diff
@@ -191,26 +192,28 @@ define `NAPALM_USERNAME` and `NAPALM_PASSWORD` in the `netbox_config` variable
...
 
-    netbox_keep_uwsgi_updated: false
+    netbox_webhooks_enabled: false
 
 Toggle `netbox_keep_uwsgi_updated` to `true` if you wish to ensure your uwsgi
 server is the latest release, otherwise uwsgi will not be updated on subsequent
 runs of your playbook.
 
-    netbox_install_epel: true
+    netbox_keep_uwsgi_updated: false
```
Comment (lae) [inline README.md]: I'm not sure if this was intentional or not, but the descriptions for role variables comes after the role variable and its default for the majority of the doc. Can you revert these changes/move the redis description below the redis variables?

File: defaults/main.yml [hunk 2]
```diff
@@ -24,10 +24,12 @@ netbox_database_port: 5432
 #netbox_database_socket: /var/run/postgresql
 
 netbox_webhooks_enabled: false
+
 netbox_redis_host: 127.0.0.1
 netbox_redis_port: 6379
 netbox_redis_password: ''
 netbox_redis_database: 0
+netbox_redis_cache_database: 1
```
Comment (lae) [inline defaults/main.yml]: I don't see this variable being used anywhere - is it necessary?

File: examples/netbox_config.yml [hunk 3]
```diff
@@ -63,6 +68,7 @@ netbox_config:
     PORT: 6379
     PASSWORD: redisisfun
     DATABASE: 0
+    CACHE_DATABASE: 1
```
Comment (lae) [inline examples/netbox_config.yml]: The `REDIS` section from the config example should be removed entirely, I think, since it's defined by other role variables now.

**Comment by lae** [comment]
Resolved in #61
